reactioncommerce/reaction
# [Marketplace] Set order(s) to a specific status

Resolves #2341 

This allows users with permission to view and process orders to be able to set/update the status of multiple orders in bulk.

#### How to test

-  Have at least 10 (preferably new) orders made in your shop
- Visit the orders page on the dashboard
- Select one order/many orders (hover over the avatar: it rolls over to a check box)

##### Testing `Bulk Actions`

- On the bar that appears at the top, click on the `Bulk Actions` button
- Select the list items in order of how they appear, and you should see the labels in the `Shipping` column change according to what you select
- Select another order/set of orders and randomly choose any order of list item (e.g. select the `Labeled` option on a new order). You should see a modal that asks if you're sure you want to skip the shipping flow


##### Testing `Capture`

- On the bar that appears at the top, click on the `Capture` button
- Observe the waiting state on the button, then the modal that shows up on success of the action
- Open the sidebar for the item selected, scroll to the `Invoice` section and observe that the payment has been captured

P.S. The `Print Invoice and Pick Sheet` step that appears on the list under `Bulk Actions` is not functional yet in this PR.
